### PR TITLE
Change VideoQueueItem to row/column dynamic layout

### LINKED ISSUE #100: Delete button on VideoQueueItem goes off the card sometimes
See screenshot
2019-09-27-154457_977x632_scrot

The video: https://www.youtube.com/watch?v=IyqUk2ZX97Q

The button is still clickable, but its mostly cut off.

### PR DESCRIPTION
**Pull request by adapap**
The thumbnail maintains the aspect ratio but gets cropped depending on screen resolution. You can change `v-img` to have the `contain` attribute in which case the full thumbnail is always visible but leaves space on either axis.

Fixes #100

### CODE REVIEW

File: src/components/VideoQueueItem.vue [hunk 1]
```diff
@@ -101,8 +97,8 @@ export default {
 		display: none;
 	}
 }
-.actions {
-	margin-left: 4px;
+.full-height {
+    height: 100%;
 }
```
Comment (dyc3) [inline src/components/VideoQueueItem.vue]: No need to make a new css class, you can use the `fill-height` property on vuetify components

File: src/components/VideoQueueItem.vue [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
+            <v-row no-gutters align="center" justify="space-between" class="full-height">
+                <v-col cols="3" class="full-height">
+                    <v-img :src="item.thumbnail" class="full-height">
+                        <span class="length">{{ videoLength }}</span>
+                    </v-img>
+                </v-col>
+                <v-col cols="8">
+                    <v-container>
+                        <v-row class="title" no-gutters> {{ item.title }}</v-row>
+                        <v-row class="description" no-gutters> {{ item.description }}</v-row>
```
Comment (dyc3) [inline src/components/VideoQueueItem.vue]: Remove the extra spaces before `{{`

**Comment by dyc3** [comment]
Also, please rebase this branch onto master and resolve the merge conflicts

**Comment by adapap** [comment]
There's probably a way to fit text equal to the height but this guarantees no overflow.

**Pull request merged**